torchvision/transforms/v2/functional/_type_conversion.py

```diff
@@ -25,3 +51,82 @@ def to_image(inpt: Union[torch.Tensor, PIL.Image.Image, np.ndarray]) -> tv_tenso
 
 to_pil_image = _F.to_pil_image
 pil_to_tensor = _F.pil_to_tensor
+
+
+@torch.jit.unused
+def to_cvcuda_tensor(pic) -> "cvcuda.Tensor":
+    """Convert a torch.Tensor to cvcuda.Tensor. This function does not support torchscript.
+
+    See :class:`~torchvision.transforms.v2.ToCVCUDATensor` for more details.
+
+    Args:
+        pic (torch.Tensor): Image to be converted to cvcuda.Tensor.
+            Tensor can be in CHW format (unbatched) or NCHW format (batched).
+            Only 1-channel and 3-channel images are supported.
```

Comment:
Should update this comment to include 2/4 channel images since we no longer filter those out with the validation function.

Reply:
@justincdavis thanks for the comments. See my answer above!